### PR TITLE
Add a bit of the homepage styling to make it more appealing

### PR DESCRIPTION
**Pull request by markgaze**

### CODE REVIEW

**Comment by markgaze** [comment]
#### 📝Changed routes:


| [**/**](https://somedevsdo-blog-brk1e9d4e-markgaze.vercel.app/) | [**/post/\[id\]**](https://somedevsdo-blog-brk1e9d4e-markgaze.vercel.app/post/[id]) |
|:-:|:-:|
|<a href="https://somedevsdo-blog-brk1e9d4e-markgaze.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://iad.microlink.io/7Bkqf5gasR7-orCMj6n9daPD6_E4qwvss9f1ZcgsKk6fqEogpbXKhukrY3vUx5Vj3kzzbMju5DkDcA28k3Cr8Q.png">(view full size)</a> |<a href="https://somedevsdo-blog-brk1e9d4e-markgaze.vercel.app/post/[id]">Screenshot of /post/[id]</a><br /><sup><a href="https://images.weserv.nl?url=https://iad.microlink.io/WPyL8wGjDxmWxUBOEBHLKwdHr4BBWOMwfzvAn-SAQmiXA7RZdcS4yRK1rSKpDW9SWUOu4cJtymMNO5Jg9RcfzA.png">(view full size)</a> |







Commit 77778b67453b4d40153cfe100ebc4cd7b8009551 (https://somedevsdo-blog-brk1e9d4e-markgaze.vercel.app).

File: pages/index.tsx [hunk 1]
```diff
@@ -3,11 +3,17 @@ import Head from "next/head";
 import Link from "next/link";
 import Layout, { siteTitle } from "../components/Layout/Layout";
 import { getSortedPostsData } from "../lib/posts";
+import ImageWithPlaceholder from "../components/ImageWithPlaceholder/ImageWithPlaceholder";
+import styles from "../styles/Home.module.scss";
+import Avatar from "../components/Avatar/Avatar";
+import getDateFormatted from "../lib/date";
 
 interface IPost {
+  category: string;
```
Comment (benmatselby) [inline pages/index.tsx:12]: Worth merging IPost with IPost in libs/posts?

Comment (markgaze) [reply pages/index.tsx:12]: Didn't see that one! Will have a look, cheers! 🙂

File: pages/index.tsx [hunk 2]
```diff
@@ -24,13 +22,40 @@ const name = "somedevsdo";
...
+              </div>
+              <h1 className={styles.title}>{postData.title}</h1>
+            </div>
+            <div className={styles.author}>
+              <Avatar border size="medium" src="/authors/benmatselby.jpg" />
+              <div className={styles.authorDetails}>
+                <p>
+                  by <strong>Ben Selby</strong>
+                </p>
+                <p>Expert</p>
```
Comment (benmatselby) [inline pages/index.tsx:48]: This silliness has to stop.

**Review by benmatselby — APPROVED** [review]

**Review by andrewbarnesweb — CHANGES_REQUESTED** [review]
Just noticed that the author and title text are changing colour, when it needs to be fixed as white

image

**Comment by markgaze** [comment]
> Just noticed that the author and title text are changing colour, when it needs to be fixed as white
> 
> image

Good spot! Fixed 🙂

**Review by andrewbarnesweb — APPROVED** [review]

**Pull request merged**